final-project/index.html

```diff
@@ -6,9 +6,306 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./dist/styles.css">
+  <!--  ICON LIBRARY -->
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
 </head>
+
 <body>
-  <h1>Hello world!</h1>
-  <script src="./dist/bundle.js"></script>
+<script src="./dist/bundle.js"></script>
```

Comment:
it is recommended to load scripts at the bottom of the document, so that users can see the site instead of a blank screen while it loads